Bubble_sort/Bubble_sort.rb

```diff
@@ -0,0 +1,26 @@
+def bubble_sort(a)
+
+    t = 1    
+    while t <= a.length - 1
+  
+      i = 0
+      while i <= a.length - t - 1
+    
+        if a[i] > a[i+1]
+          temp = a[i]
+          a[i] = a[i+1]
+          a[i+1] = temp
```

Comment:
全然マストではないですがrubyにはこう言った書き方もあるというご紹介です。
```suggestion
          a[i], a[i+1] = a[i+1], a[i]
```